genkeys.go

```diff
@@ -70,6 +73,7 @@ func init() {
 		"RightBracket": {"BracketRight"},
 		"GraveAccent":  {"Backquote"},
 		"NumLock":      {"NumLock"},
+		"Menu":         {"ContextMenu"},
```

Comment:
Ah sorry, could you move all the new keys here? I think this is my last request.

Reply:
nvm, you did the right work :-)